consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/RoundTimer.java

```diff
@@ -45,7 +45,12 @@ public RoundTimer(
     this.queue = queue;
     this.bftExecutors = bftExecutors;
     this.currentTimerTask = Optional.empty();
-    this.baseExpiryMillis = baseExpirySeconds * 1000;
```

Comment:
If the baseExpiryMillis type was changed to a duration, then we could handle both milliseconds and seconds without any special logic